packages/background/src/controllers/AccountTrackerController.ts

```diff
@@ -1053,6 +1053,19 @@ export class AccountTrackerController extends BaseController<AccountTrackerState
         this.emit(AccountTrackerEvents.CLEARED_ACCOUNTS);
     }
 
+    /**
+     * Resets an account and associated balances
+     *
+     */
+    public resetAccount(address: string): void {
+        const stateAccounts = this.store.getState().accounts;
+        stateAccounts[address].balances = {};
```

Comment:
Shouldn't we add Blank Token balance again? @julianariel

Reply:
That makes sense 👍

Reply:
I've just tested it, works correctly 🚀